pkg/cloud/cloud_test.go

```diff
@@ -596,3 +596,213 @@ func TestDescribeFileSystem(t *testing.T) {
 		t.Run(tc.name, tc.testFunc)
 	}
 }
+
+func TestDescribeMountTargets(t *testing.T) {
```

Comment:
wdyt about refactoring to be table driven, I want to stop the "testFunc" pattern, it is verbose and harder to read in my opinion compared to providing input, output, expected error

Reply:
Yeah I think table driven approach reduces the verbose nature of the these tests. I'll refactor these tests.